dotnetv3/S3/RestoreArchivedObjectExample/RestoreArchivedObjectExample/RestoreArchivedObject.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace RestoreArchivedObjectExample
+{
+    using System;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    /// <summary>
+    /// This example shows how to restore an archived object in an Amazon
+    /// Simple Storage Service (Amazon S3) bucket. The example was created
+    /// using the AWS SDK for .NET version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class RestoreArchivedObject
+    {
+        public static void Main()
+        {
+            string bucketName = "doc-example-bucket";
+            string objectKey = "archived-object.txt";
+
+            // Specify your bucket region (an example region is shown).
+            RegionEndpoint bucketRegion = RegionEndpoint.USWest2;
+
+            IAmazonS3 client = new AmazonS3Client(bucketRegion);
+            RestoreObjectAsync(client, bucketName, objectKey).Wait();
+        }
+
+        /// <summary>
+        /// This method restores a deleted object from an S3 bucket.
```

Comment:
Do it restore and 'archived' or 'deleted' bucket? Possibly no difference, but we should be consistent

Reply:
Should be restores an archived object.